### DISCUSSION TITLE
"Broker" List - List of sellers / buyers

### DISCUSSION THREAD

**Discussion by Q4FOIL** [sunflower-land/sunflower-land#1234] [Ideas]
After the introduction of the system for the sale of goods in the **Goblin Village**, I would say to create a modal where all those who have started "to sell" appear so as to make it **easier for the seller and the buyer to meet**

FIRST CONCEPT 

Concept : 

![Schermata 2022-07-14 alle 13 40 54](https://user-images.githubusercontent.com/58088988/178974448-fe1cb517-4013-4f98-b98e-ff0084a20acf.png)
.

Second Concept [HERE](https://github.com/sunflower-land/sunflower-land/discussions/1234#discussioncomment-3162229)

Tell me what you think :P 

- **Q**uatrefoil
- Farm ID : **122927**

**Comment by adamhannigan** [comment]
Love it! @Q4FOIL There are actually a few open source developers building this out right now. I have even seen a few websites already launched for this. 

All of the data is public so hopefully a contributor adds this into the game ;)

**Reply by Q4FOIL** [reply]
> [<img alt="" width="32" height="32" src="https://avatars.githubusercontent.com/u/11745561?s=64&amp;v=4"> adamhannigan ](/adamhannigan)

Hello @adamhannigan,

Not everyone has the concept of open source, as not everyone can manage external data.

The external site is very convenient for the real "skilled", while this UI could benefit those who want to stay in-game in any case without having to make external movements.

I think getting out of Sunflower Land may be a mistake, but that's just my opinion :)

Glad to contribute, that's not all;)

- Quatrefoil
- Farm ID 122927

**Comment by Q4FOIL** [comment]
1/ Update

This is my second idea


![Schermata 2022-07-16 alle 16 04 55](https://user-images.githubusercontent.com/58088988/179358230-fdd5dfd6-9061-49d7-b3d8-3db6b1b6b65d.png)


@adamhannigan tell me if it's better

**Reply by Polysemouse** [reply]
Another suggestion is to follow the UI implementation of the Trove marketplace (https://trove.fandom.com/wiki/Player_Marketplace).  Since Trove's market is quite similar to Sunflower Land (sellers create orders and buyers buy orders from the available sell orders), something similar can be implemented in Sunflower Land in terms of the UI

![image](https://user-images.githubusercontent.com/107602352/180220872-96a67368-2092-4db8-afb7-81414b6a1983.png)

**Reply by Q4FOIL** [reply]
> Another suggestion is to follow the UI implementation of the Trove marketplace (https://trove.fandom.com/wiki/Player_Marketplace). Since Trove's market is quite similar to Sunflower Land (sellers create orders and buyers buy orders from the available sell orders), something similar can be implemented in Sunflower Land in terms of the UI
> 
> ![image](https://user-images.githubusercontent.com/107602352/180220872-96a67368-2092-4db8-afb7-81414b6a1983.png)

Hello @Polysemouse 

I think it's too hard to introduce this UI for Trading, anyway it's a good idea but i think unrealizable 

@adamhannigan

**Reply by Yougottabekiddingme123** [reply]
> Another suggestion is to follow the UI implementation of the Trove marketplace (https://trove.fandom.com/wiki/Player_Marketplace). Since Trove's market is quite similar to Sunflower Land (sellers create orders and buyers buy orders from the available sell orders), something similar can be implemented in Sunflower Land in terms of the UI
> 
> ![image](https://user-images.githubusercontent.com/107602352/180220872-96a67368-2092-4db8-afb7-81414b6a1983.png)

I love this. I think this could greatly improve the  trading experience.  Trading resources in many games provides half the fun as players seek to buy/sell/trade regularly. A reduction from 10% to 5% would assist in this regard but  a more user friendly marketplace would also add value to the game. Unfortunately, I dont think that changes to the marketplace will be prioritized until after land expansion. I think the devs are satisfied with their super basic marketplace that we currently have and are now focused in land expansion. This is understandable although not ideal...
